### ISSUE TITLE
WMR Samsung Odyssey OpenVR can't be installed. 

### ISSUE THREAD

**Issue by mena99** [ValveSoftware/steamvr_unity_plugin#780]
Hello,

I've tried it on version 2019.3.7f and 2020.1.0BETA

**How did I Install the package :**

Package manager > Add package from disk...
The scene was  just a plane with "Room-scale XR-Rig" 
* no other assets was in the scene 
* windows 10 + Samsung Odyssey (Windows mixed reality )

**2019.3.7f**

After enabling the OpenVR loader and hitting the play button

SteamVR was launched to run the Scene ....

workaround it was me adding Windows mixed reality loader underneath it and that ran the scene on windows mixed reality

**2020.1.0Beta**
so what I have done was that I downloaded the package  
while trying to Enabling the OpenVr Loader I got this error 

`_Package Metadata store error. Check if there are any other errors in the console and make sure they are corrected before trying again.
 Failure reason: Error installing package com.valve.openvr. Error Code: Failure Error Message: Unable to add package [com.valve.openvr]:
  Package [com.valve.openvr] cannot be found
UnityEditor.EditorApplication:Internal_CallUpdateFunctions()_
`

again I was able to run the scene with windows mixed reality loader.

* many times after I save the scene and relaunch it, even WMR stops from working, always runs on steamVR *

**Comment by SWinxy** [comment]
Having the same issue. Trying to install OpenVR Loader, but get this error:
> Package Metadata store error. Check if there are any other errors in the console and make sure they are corrected before trying again.
> Failure reason: Error installing package com.valve.openvr. Error Code: Failure Error Message: Unable to add package [com.valve.openvr]:
> Package [com.valve.openvr] cannot be found
> UnityEditor.EditorApplication:Internal_CallUpdateFunctions()

**Comment by mena99** [comment]
> Having the same issue. Trying to install OpenVR Loader, but get this error:
> 
> > Package Metadata store error. Check if there are any other errors in the console and make sure they are corrected before trying again.
> > Failure reason: Error installing package com.valve.openvr. Error Code: Failure Error Message: Unable to add package [com.valve.openvr]:
> > Package [com.valve.openvr] cannot be found
> > UnityEditor.EditorApplication:Internal_CallUpdateFunctions()

what device are you using? WMR?

**Comment by zite** [comment]
Try going to the package manager and uninstalling the OpenVR XR Plugin. Then reinstall with the latest release from the releases page.

**Comment by ll-M-ll** [comment]
> Try going to the package manager and uninstalling the OpenVR XR Plugin. Then reinstall with the latest release from the releases page.

I have tried that, few days ago, all preview tools disappeared including the ( OPENVR LOADER) ... I had to go back to the earlier version to be able to use it.

**Comment by zite** [comment]
I'm not sure I understand. You tried uninstalling the version you had, and reinstalling using the Installer package at https://github.com/ValveSoftware/unity-xr-plugin/releases/tag/installer and the OpenVR Loader option didn't show up in the XR Management settings?

**Comment by ll-M-ll** [comment]
> I'm not sure I understand. You tried uninstalling the version you had, and reinstalling using the Installer package at https://github.com/ValveSoftware/unity-xr-plugin/releases/tag/installer and the OpenVR Loader option didn't show up in the XR Management settings?

yeah I went to the package manager upgraded the XR to the latest I think it was x.x.x3 I don't remember , when I did that the XR tools ( preview ) disappeared from the package manager ( both of them ) so I thought maybe they weren't compatible with each other, also the OpenVR loader disappeared ... I tried to go back to the older version unfortunately it didn't work either so I've went back to my backup file and started the whole process from there and this time without upgrading anything.
